helm/kubernetes.go

```diff
@@ -34,13 +33,16 @@ var tillerTunnel *kube.Tunnel
 
 //GetK8sConnection creates a new Kubernetes client
 func GetK8sConnection(kubeConfig []byte) (*kubernetes.Clientset, error) {
+	var client *kubernetes.Clientset
 	config, err := GetK8sClientConfig(kubeConfig)
 	if err != nil {
 		return nil, fmt.Errorf("create kubernetes config failed: %v", err)
 	}
-	client, err := kubernetes.NewForConfig(config)
-	if err != nil {
-		return nil, fmt.Errorf("create kubernetes connection failed: %v", err)
+	for i := 0; i < 2; i++ {
```

Comment:
please write a comment why we need to get the client for two times without an error.